mirroring/action.yaml

```diff
@@ -0,0 +1,17 @@
+name: Publish Docker Image
+on:
+  release:
+    types:  [published]
+jobs:
+  name: Push Docker image to Docker Hub
+  runs-on: apline-latest
+  steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+    - name: Push to Docker Hub
+      uses: docker/build-push-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        repostiory: a0su/sigstore_mirroring
```

Comment:
nit: repository has a typo here, could this be problematic?

Reply:
Ah yes thank you for fixing that. I didn't even catch that beforehand.